just_audio/ios/Classes/Extensions/DarwinEqualizerMessage.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  DarwinEqualizerMessage.swift
+//  just_audio
+//
+//  Created by Mac on 24/09/22.
+//
+import kMusicSwift
+
+extension [String: Any?] {
+    /// returns an equalizer with an activated preset
+    var equalizer: Equalizer? {
+        get throws {
+            if self["type"] as? String != "DarwinEqualizer" {
+                return nil
```

Comment:
maybe here it's better to throw something, so then we can guarantee an `Equalizer` that is never nil

Reply:
Non sempre è necessario creare l' equalizzatore, forse possiamo rinominare il metodo in equalizerOrNull